### ISSUE TITLE
[BITV]: `focus-visible` is missing on `NcActionLink` if an element have `active` class

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#43202]
focus-visible is missing on NcActionLink if an element have active class **and** actions are opened via mouse.

![Screenshot from 2024-01-29 10-14-28](https://github.com/nextcloud/server/assets/6078378/c944ab1e-3917-45ab-a431-1a425f3751f6)

`focus-visible` on `NcActionLink` action element have to be the same as for `NcActionButton` action element, like:

![Screenshot from 2024-01-30 11-16-17](https://github.com/nextcloud/server/assets/6078378/a93f1dec-7a10-4ca5-bbec-e7f8011d09e1)

**Comment by szaimen** [comment]
![image](https://github.com/nextcloud/server/assets/42591237/6f83bba9-655f-46a3-8aa1-324f0473c9a8)
this looks broken as well?

**Comment by JuliaKirschenheuter** [comment]
> ![image](https://private-user-images.githubusercontent.com/42591237/300771277-6f83bba9-655f-46a3-8aa1-324f0473c9a8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY2MTA2MzYsIm5iZiI6MTcwNjYxMDMzNiwicGF0aCI6Ii80MjU5MTIzNy8zMDA3NzEyNzctNmY4M2JiYTktNjU1Zi00NmEzLThhYTEtMzI0ZjA0NzNjOWE4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTMwVDEwMjUzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTczYmNmYzFhODcxMDA3OWE5NTA2NzM0NjhiZjUyM2FjNzcxOTE1NWFkNDIzMWZkYmIyMWUxNmJkYjcwNjMwNjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.CwqVaT1kvgUfkzE40-odXPiGYesfrAdPFENgCxHQzTY) this looks broken as well?

this is another issue https://github.com/nextcloud/viewer/issues/2138

**Comment by JuliaKirschenheuter** [comment]
After investigation with @ShGKme come to solution that this is a browser bug: browser (in my specific case Chrome) is not able to recognize that a `focus-visible` was set via keyboard on a link. This is very browser and version depending and actually it works if a user uses a keyboard to focus into NcActions button. So, closing an issue

**Comment by ShGKme** [comment]
Could be related to the issue on Safari, where we have the opposite - a browser always thinks that menu was opened via keyboard
